IllyaBranch

IllyaPriymak opened the pull request:
Home Work

Commit:
initial commit

kate-sagittarius commented on index.html:
```diff
@@ -5,9 +5,9 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="styles/index.css">
-    <title>Document</title>
+    <title>My resume</title>
```
you shouldn't be changing this file

kate-sagittarius commented on team-members/IllyaPriymak.html:
```diff
@@ -0,0 +1,81 @@
+    </header>
+    <main>
+      <section>
+        <p>Hello. My name is Illya, i am 20 years old and i live in Vishneve,
```
content of p should be placed from the new line

kate-sagittarius commented on team-members/IllyaPriymak.html:
```diff
@@ -0,0 +1,81 @@
+        <p>Hello. My name is Illya, i am 20 years old and i live in Vishneve,
+           im working barista, and i am students in Beetroot Academy</p>
+        <br>
+        <h2><section id="part2"><i>Positive qualities about myself</i></h2>
```
section shouldn't be placed inside h1 + it has no closing tag

kate-sagittarius commented on team-members/IllyaPriymak.html:
```diff
@@ -0,0 +1,81 @@
+   E-mail: <input type="email" name="email" autocomplete="off"><br>
+   <input type="submit">
+ </section>
+</form>
```
mind the indents

kate-sagittarius reviewed (changes requested):
please, check this file and correct indents, remove section tags from h1, h2 etc tags

Commit:
you shouldn't be changing this file
content of p should be placed from the new line
section shouldn't be placed inside h1 + it has no closing tag
mind the indents

kate-sagittarius reviewed (approved):
good job!

kate-sagittarius committed:
Merge branch 'master' into HEAD
